tests/unit/test_serialization.py

```diff
@@ -59,6 +59,7 @@ async def test_deserialization_fails(default_config_path, mock_registry_client):
         await deserializer.deserialize(invalid_header_payload)
 
     # for now we ignore the packed in schema id
+    # This raises UnicodeDecodeError in avro.io.read_utf8
```

Comment:
Is this a left over comment? the test is not caughting this exception

Reply:
Left over, clarifications for self when testing the differences with Aiven fork and Avro versions.
Will remove.